gcloud-java-compute/src/test/java/com/google/gcloud/compute/it/ITComputeTest.java

```diff
@@ -447,4 +446,190 @@ public void testListZonesWithFilter() {
     assertEquals(ZONE, zoneIterator.next().zoneId().zone());
     assertFalse(zoneIterator.hasNext());
   }
+
+  @Test
+  public void testListGlobalOperations() {
```

Comment:
Is the reason there aren't integration tests for getting/deleting operations because it's unreliable to create and then get/delete an operation?  Or do you think we can test this with a timeout in a fashion that won't be too flaky?